tests/Database/DatabaseEloquentBelongsToManyTest.php

```diff
@@ -116,7 +116,7 @@ public function testModelsAreProperlyMatchedToParents()
         $this->assertEquals(2, $models[1]->foo[0]->pivot->user_id);
         $this->assertEquals(2, $models[1]->foo[1]->pivot->user_id);
         $this->assertEquals(2, count($models[1]->foo));
-        $this->assertEquals(0, count($models[2]->foo));
+        $this->assertEquals(0, count((array) $models[2]->foo));
```

Comment:
I think the problem here doesn't actually lie with Laravel perhaps, since this is a BC break in PHP, rather than being a break that Laravel has made? Would be interested to hear other opinions, or other responses to this break across other projects.